hasura/migrations/default/1676345211557_org_members/up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE public.org_members (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
This column is probably unnecessary, but its not a big deal

Reply:
it's a bit more future-proof -- we'll need it if we ever want to have something have a foreign key to this table